openmls/src/framing/validation.rs

```diff
@@ -186,16 +186,18 @@ impl DecryptedMessage {
             // Preconfigured senders are not supported yet #106/#151.
             Sender::Preconfigured(_) => unimplemented!(),
             Sender::NewMember => {
-                // Since this allows only commits to have a sender type `Member`, it checks
+                // Since this allows only commits or external Add proposals to have a sender type `NewMember`, it checks
                 // ValSem112
-                if let MlsPlaintextContentType::Commit(commit) = self.plaintext().content() {
-                    if let Some(path) = commit.path() {
-                        Ok(path.leaf_key_package().credential().clone())
-                    } else {
-                        Err(ValidationError::NoPath)
+                // ValSem113
```

Comment:
I kinda fail to understand, but is the following block performing both ValSem112 AND ValSem113?

Maybe replace it by 
```suggestion
                // ValSem112 & ValSem113
```

so that it's a tad clearer for people who look at the code for the first time